Jenkinsfile

```diff
@@ -271,6 +271,9 @@ pipeline {
         string(name: 'CI_UNIT_VM1_LABEL',
                defaultValue: 'ci_vm1',
                description: 'Label to use for 1 VM node unit and RPM tests')
+        string(name: 'CI_UNIT_VM1_NVME_LABEL',
+               defaultValue: 'bwx_vm1',
```

Comment:
In a future pass, I am thinking I will split the nvme ones into another run_test script altogether.  Also thinking of rewriting that script in python.  But I want to get this in to prevent any regressions while we work out any additional details

Reply:
We will eventually check this feature branch into master but I suspect we will resolve this issue before such a time.

Reply:
I have created a ci_ssd_vm1 label in CI to indicate that there is a /dev/vdb ssd drive on them.
On the newer KVM host distros,  There is a way to provide a emulated NVMe drive so did not want to use nvme in the label.